examples/camera-app/linux/src/clusters/webrtc-provider/webrtc-provider-manager.cpp

```diff
@@ -306,9 +306,6 @@ CHIP_ERROR WebRTCProviderManager::HandleProvideAnswer(uint16_t sessionId, const
 
     mPeerConnection->setRemoteDescription(sdpAnswer);
 
-    MoveToState(State::SendingICECandidates);
```

Comment:
Based on the spec, 11.4.3.2. Battery Camera in Standby Flow, the camera need to send ICE candidates to camera-controller after handling ProvideAnswer, why this step is removed?

Reply:
Thank you for your review. The primary objective of this PR is to support TC-WEBRTC-1.1 using YAML Test case https://github.com/project-chip/connectedhomeip/pull/38407, If ice-candidate exchange is within some automated flow, it is not possible for the YAML TC framework to detect such command reception/invocation. Hence, I have separated out the automatic ice candidate exchange after SDP exchange, so that it can triggered manually via webrtc provide-ice-candidates and also when required by the YAML TC.

Reply:
this should break the current CI check since the WebRTC session establishment will never succeed, no? Have you rebase/merge your PR with the latest master?